modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_composed.mustache

```diff
@@ -34,9 +28,14 @@
         self._composed_instances = composed_info[0]
         self._var_name_to_model_instances = composed_info[1]
         self._additional_properties_model_instances = composed_info[2]
+        unused_args = composed_info[3]
 
 {{#requiredVars}}
         self.{{name}} = {{name}}
 {{/requiredVars}}
         for var_name, var_value in six.iteritems(kwargs):
+            if var_name in unused_args and self._configuration is not None and \
+                            self._configuration.discard_unknown_keys:
```

Comment:
How about adding `and not self._additional_properties_model_instances`?
That way when we have _additional_properties_model_instances we will assign those additional props.

Reply:
done